LOHHLAscript.R

```diff
@@ -578,26 +671,27 @@ if(mapping.step){
     regionDir <- paste(workDir, '/', BAMid, sep = '')
```

Comment:
Seems to only work when input files are in the run directory due to how this is defined.